docs/man/monerod.1

```diff
@@ -0,0 +1,5343 @@
+.TP
+\fB\--test-drop-download\fR
+For net tests: in download, discard ALL blocks instead checking/saving them (very fast)
+.TP
+\fB\--test-drop-download-height\fR arg (=0)
+Like test-drop-download but discards only after around certain height
+.TP
+\fB\--keep-fakechain\fR
+Don't delete any existing database when in fakechain mode.
+.TP
+\fB\--regtest\fR
+Run in a regression testing mode.
+.TP
+\fB\--fixed-difficulty\fR arg (=0)
+Fixed difficulty used for testing.
+
+
+.SS Legacy Options
+.TP
+\fB\--ban-list\fR arg
+Specify ban list file, one IP address per line
+.TP
+\fB\--enable-dns-blocklist\fR      
+Apply realtime blocklist from DNS
+.TP
+\fB\--zmq-pub\fR arg
+Address for ZMQ pub - tcp://ip:port or ipc://path
+.TP
+\fB\--zmq-rpc-bind-ip\fR arg (=127.0.0.1)
+IP for ZMQ RPC server to listen on
+.TP
+\fB\--zmq-rpc-bind-port\fR arg (=18082, 28082 if 'testnet', 38082 if 'stagenet')
+Port for ZMQ RPC server to listen on
+.TP
+\fB\--fluffy-blocks\fR
+Relay blocks as fluffy blocks (obsolete, now default)
+.TP
+\fB\--no-fluffy-blocks\fR
+Relay blocks as normal blocks
+
+
+.SH Daemon commands:
+Commands give access to specific services provided by the daemon. Commands are executed against the running daemon.
+.br
+Their names follow the \fBcommand_name\fR pattern.
+
+
+.SS Info:
+.TP
+\fBhelp [<command>]\fR
+Show the help section or the documentation about a <command>.
+.TP
+\fBapropos <keyword> [<keyword> ...]\fR
+Search all command descriptions for keyword(s).
+.TP
+\fBversion\fR
+Print version information.
+.TP
+\fBstatus\fR
+Show the current status.
+.TP
+\fBprint_net_stats\fR
+Print network statistics.
+
+
+.SS P2P Network:
+.TP
+\fBprint_pl [white] [gray] [pruned] [publicrpc] [<limit>]\fR
+Print the current peer list.
+.TP
+\fBprint_pl_stats\fR
+Show the full peer list statistics (white vs gray peers). White peers are any peers your node has completed a handshake with. Grey nodes are nodes that your node hasn’t tried to connect to yet.
+.TP
+\fBprint_cn\fR
+Show connected peers with connection initiative (incoming/outgoing) and other stats.
+.TP
+\fBban [<IP>|@<filename>] [<seconds>]\fR
+Ban a given <IP> or list of IPs from a file for a given amount of <seconds>. By default the ban is for 24h.
+.TP
+\fBbans\fR
+.br
+Show the currently banned IPs.
+.B Example: 187.63.135.161 banned for 86397 seconds
+.TP
+\fBbanned <address>\fR
+Check whether an <address> is banned.
+.TP
+\fBunban <address>\fR
+Unban a given <IP>.
+.TP
+\fBin_peers <max_number>\fR
+Set the <max_number> of incoming connections from other peers.
+.TP
+\fBout_peers <max_number>\fR
+Set the <max_number> of outgoing connections to other peers.
+.TP
+\fBlimit [<kB/s>]\fR
+Get or set the download and upload limit.
+.TP
+\fBlimit_down [<kB/s>]\fR
+Get or set the download limit.
+.TP
+\fBlimit_up [<kB/s>]\fR
+Get or set the upload limit.
+
+
+
+.SS Transaction pool:
+.TP
+\fBflush_cache [bad-txs] [bad-blocks]\fR
```

Comment:
I wouldn't consider `flush_cache bad-blocks` a transaction pool command; would suggest moving `flush_cache` to the blockchain section.